.azure/azure-pipelines.ci.yml

```diff
@@ -359,3 +359,16 @@ stages:
         arguments: -Branch $(Build.SourceBranch)
       env:
         AzDO_PAT: $(AzDO_PAT)
+
+#
+# Distribution
+#
+
+- stage: distribution
+  displayName: Distribution
+  dependsOn:
+  - build_windows
+  - build_linux
+  - build_winkernel
```

Comment:
No need to worry about kernel mode stuff.

Reply:
Remove the dependency on build_winkernel too; and all the winkernel references in the distribution script.